windows-build/deploy-binary.bat

```diff
@@ -0,0 +1,5 @@
+docker run --name bubblesub-deploy bubblesub
+docker cp bubblesub-deploy:C:\bubblesub\dist\ .
+docker rm bubblesub-deploy
+7z a bubblesub.tar dist\windows\bubblesub.exe
+7z a bubblesub.tar.gz bubblesub.tar
```

Comment:
We're doing windows builds, so let's release it in a `.zip`…
unless you have something else in mind that requires the packages to get shipped as `.tar.gz`?

Reply:
Ok, I will use a `.zip` file then